index.html

```diff
@@ -702,49 +702,80 @@
             <pre class="prettyprint linenums">
 &lt;input type=&quot;submit&quot; value=&quot;press me&quot; class=&quot;button button-pill button-primary&quot;/&gt;
 &lt;button class=&quot;button button-pill button-primary&quot;&gt;press me&lt;/button&gt;
-
 &lt;!-- DISABLED BUTTONS --&gt;
 &lt;input  disabled type=&quot;submit&quot;  value=&quot;press me&quot; class=&quot;button button-pill button-primary&quot;/&gt;
 &lt;button disabled class=&quot;button  button-pill button-primary&quot;&gt;press me&lt;/button&gt;
 &lt;a href=&quot;#&quot; class=&quot;button disabled button-pill button-primary&quot;&gt;press me&lt;/a&gt;</pre>
         </section>
 
+
+        <section id="form" class="showcase">
+            <hgroup>
+                <h3>Modern Button</h3>
+            </hgroup>
+
+            <section class="gallery">
+
+                <ul>
+                    <li><input type="submit" value="press me" class="modern-button" /></li>
+                    <li><button class="modern-button">press me</button>  </li>
```

Comment:
there's no `.modern-button` CSS so I'm not sure what you were "adding" exactly. I appreciate the effort, but I'm going to have to close this PR as I don't see anything added. Also, for a visual / css library, you would want to post screenshots for a new widget.